Classes/Emogrifier.php

```diff
@@ -1529,18 +1536,32 @@ private function getNodesToExclude(\DOMXPath $xPath)
     public function handleXpathError($type, $message, $file, $line, array $context)
     {
         if ($type === E_WARNING && isset($context['cssRule']['selector'])) {
-            throw new \InvalidArgumentException(
-                sprintf(
-                    '%s in selector >> %s << in %s on line %s',
-                    $message,
-                    $context['cssRule']['selector'],
-                    $file,
-                    $line
-                )
-            );
+            if ($this->debug) {
```

Comment:
We don't need the else here as throwing an exception will exit the method anyway.